sktime/annotation/ggs.py

```diff
@@ -307,34 +355,50 @@ def find_change_points(self, data: npt.ArrayLike) -> List[int]:
 
         return change_points
 
-    def predict(self, X: npt.ArrayLike):
-        """Predict."""
-        self.change_points_ = self.find_change_points(X)
-
-        labels = np.zeros(X.shape[0], dtype=np.int32)
-        for i, (start, stop) in enumerate(
-            zip(self.change_points_[:-1], self.change_points_[1:])
-        ):
-            labels[start:stop] = i
-        return labels
-
 
 class GreedyGaussianSegmentation(BaseEstimator):
     """Greedy Gaussian Segmentation Estimator.
 
+    The method apprixmates solutions for the problem of breaking a
+    multivariate time series into segments, where the data in each segment
+    could be modeled as independent samples from a multivariate Gaussian
+    distribution. It uses a dynamic programming search algorithm with
+    a heuristic that allows finding approximate solution in linear time with
+    respect to the data length and always yields locally optimal choice.
+
```

Comment:
This comment is not meant to be blocking - but I think what would push these docs to the next level would be a simple breakdown of the strengths (eg - no need for prior knowledge of the values of the means/sds, quick run time) and some of the drawbacks (knowledge of k, reliance on Gaussian assumptions etc) of this estimator in simple language.  Or even better - what types of problems does this algorithm work well with?  

I think having this sort of information readily available would make it so much easier for someone comparing the different annotation algorithms to have an idea of which one they actually need.

Reply:
I :100: % agree, that would be great to have. Right now it's hard to source that information since actual studies comparing these methods and  are not out yet. It would be great to come up with such a comparison but than it would probably be better suited to be share as a user guide in a separate doc/notebook rather than docstrings.